docs/tre-admins/auth.md

```diff
@@ -1,30 +1,32 @@
 # Authentication and authorization
 
-[Azure Active Directory (AAD)](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis) is the backbone of Authentication and Authorization in the Trusted Research Environment.
-
-AAD holds the identities of all the TRE/workspace users, including administrators, and connects the identities with applications which define the permissions for each user role.
+[Azure Active Directory (AAD)](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis) is the backbone of Authentication and Authorization in the Trusted Research Environment. AAD holds the identities of all the TRE/workspace users, including administrators, and connects the identities with applications which define the permissions for each user role.
 
 ## Pre-requisites
-The following values are needed to be in place to run the script. (`/templates/core/.env`)
+The following values are needed to be in place before you run the creation process. (`/templates/core/.env`)
```

Comment:
is there some pre-amble missing before we even get here?

For example, before **what** creation process? Why will I be running this process?

All it needs is some additional introduction to say that...

- there's an automated process of setting up the basics of the AzureTRE AAD
- that script read and then uses environment variables

... and then "The following values are needed...." then follows